wp-content/themes/core/integrations/gravity-forms/css/label.pcss

```diff
@@ -4,6 +4,11 @@
 
 .gfield_label {
 	@mixin form-label;
+
+	/* CASE: Theme Light */
+	.t-theme--light & {
+		color: var(--color-white);
+	}
```

Comment:
Similar to what we do in other typographic mixins, we could potentially just add this to the `form-label` mixin so it'd work everywhere its used?